examples/01_plotting/plot_surf_stat_map.py

```diff
@@ -120,10 +124,34 @@
 
 from nilearn import plotting
 
-plotting.plot_surf_roi(fsaverage['pial_left'], roi_map=pcc_map,
-                       hemi='left', view='medial',
-                       bg_map=fsaverage['sulc_left'], bg_on_data=True,
-                       title='PCC Seed')
+plotting.plot_surf_roi(
+    fsaverage['pial_left'],
+    roi_map=pcc_map,
+    hemi='left',
+    view='medial',
+    bg_map=fsaverage['sulc_left'],
+    bg_on_data=True,
+    title='PCC Seed'
+)
+
+###############################################################################
+# Using a flat mesh can be useful in order to easily locate your area
+# of interest on the cortex. To make this plot easier to read,
+# we use the mesh curvature as a background map.
+# Rescaling the background map yields nicer plots.
+
+bg_map = np.sign(surface.load_surf_data(fsaverage['curv_left']))
+bg_map_rescaled = (bg_map + 1) / 4 + 0.25
```

Comment:
Can we avoid this cryptic thing in the example ?

Reply:
I think we have several options:
1. remove the cryptic line ; the thumbnail will not be as nice looking but I can live with that :blush: 
2. hide this cryptic computation in a util function with an explicit name, for instance `plotting.scale_surf_bg_map()`. By default it would scale a map to have values between [0.25, 0.75] (which is what the cryptic line does) but users could set these limits. This could fit within this PR I guess.
3. ship pre-computed background maps as @jeromedockes once suggested. This can come in addition to 2. in an other PR (I think it needs extensive testing). This would only work for fsaverage meshes, and users plotting surfaces on custom meshes would still benefit from util functions computing the nicest background maps for them.

Reply:
Maybe we're overdoing it. Could just keep it with a more explicit comment explaining why we have to do this.

Reply:
Sure! That's what I tried to do line 141, but maybe a comment in the code would be more helpful.

Reply:
Could the comment be more explicit: "we change the background map range from [-1, 1] to [.27, .75]".

Reply:
That's roughly what the new comment I uploaded yesterday is saying:
```python
###############################################################################
# Using a flat mesh can be useful in order to easily locate your area
# of interest on the cortex. To make this plot easier to read,
# we use the mesh curvature as a background map.

bg_map = np.sign(surface.load_surf_data(fsaverage['curv_left']))
# np.sign yields values in [-1, 1]. We rescale the background map
# such that values are in [0.25, 0.75], resulting in a nicer looking plot.
bg_map_rescaled = (bg_map + 1) / 4 + 0.25
```
I don't know why the interface no longer indicates that the line you commented is outdated.